api-module-library/hubspot/manager.js

```diff
@@ -41,8 +42,8 @@ class Manager extends ModuleManager {
                 instance.entity.credential
             );
             instance.credential = credential;
-            apiParams.access_token = credential.accessToken;
-            apiParams.refresh_token = credential.refreshToken;
+            apiParams.access_token = credential.access_token;
```

Comment:
Also related to a breaking change. I think we should start following a standardized naming convention for the names, instead of mixing camel case and snake case. I'm in favor of changing `apiParams.access_token` to `apiParams.accessToken` instead of changing the credential variables (see comment below).